### DISCUSSION TITLE
Quick Citation Format of Zotero with TeXstudio

### DISCUSSION THREAD

**Discussion by RabahHouna** [retorquere/zotero-better-bibtex#2531] [Q&A] [answered]
I use TeXstudio 4.5.2 and Zotero 6.0.26.
I have a question : 
Is the script Launch Zotero Picker : 
%SCRIPT
var url = 'http://127.0.0.1:23119/better-bibtex/cayw??format=biblatex&command=autocite&texstudio=true&minimize=true';
system('curl -s -A "TeXstudio" ' + url);
do not show the pop-up window on the front?
And how I can do that, I mean how to show it in the front?

**Comment by retorquere** [comment, marked as answer]
Does this help? https://forums.zotero.org/discussion/comment/389917/#Comment_389917

**Reply by RabahHouna** [reply]
Yes, thank very very very much. It is almost one month or more I search and ask for a solution to this.
